deployment/k8s-config/kustomize/base/arc/arc-sshd-deployment.yaml

```diff
@@ -15,6 +15,13 @@ spec:
       labels:
         run: arc-sshd
     spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: nvidia.com/gpu.count
+                operator: DoesNotExist
```

Comment:
priority class name should be added to this spec (and to others that don't have it already)

Reply:
Added and verified.